Libplanet.Net/Consensus/PreVoteState.cs

```diff
@@ -44,11 +45,7 @@ public class PreVoteState<T> : IState<T>
 
             roundContext.ResetVote();
```

Comment:
I think this is not required because it became to distinguish votes by its certain property?